experiments/5.0-je-v1-streamlistener.ipynb

```diff
@@ -0,0 +1,389 @@
+       "\n",
+       "                           name  \\\n",
+       "0      Quesnel Cariboo Observer   \n",
+       "1  Verdant Square Network DC-MD   \n",
+       "2                      Chris 🇺🇸   \n",
+       "3          Armenian News-NEWSam   \n",
+       "4             The Hope Standard   \n",
+       "\n",
+       "                                               tweet  replies_count  \\\n",
+       "0  B.C. orchardists face challenges from COVID-19...              0   \n",
+       "1  Coronavirus Latest: Johns Hopkins Confirms Mor...              0   \n",
+       "2  Chicago area food banks go into 'disaster mode...              0   \n",
+       "3  #EU to provide #EUR 51,000,000 to #Armenia to ...              0   \n",
+       "4  B.C. orchardists face challenges from COVID-19...              0   \n",
+       "\n",
+       "   retweets_count  likes_count  \n",
+       "0               0            0  \n",
+       "1               0            0  \n",
+       "2               0            0  \n",
+       "3               0            0  \n",
+       "4               0            0  "
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df = pd.read_csv(\"data_20200402.csv\"); df.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<class 'pandas.core.frame.DataFrame'>\n",
+      "RangeIndex: 417 entries, 0 to 416\n",
+      "Data columns (total 9 columns):\n",
+      " #   Column          Non-Null Count  Dtype \n",
+      "---  ------          --------------  ----- \n",
+      " 0   created_at      417 non-null    object\n",
+      " 1   id              417 non-null    int64 \n",
+      " 2   user_id         417 non-null    int64 \n",
+      " 3   username        417 non-null    object\n",
+      " 4   name            417 non-null    object\n",
+      " 5   tweet           417 non-null    object\n",
+      " 6   replies_count   417 non-null    int64 \n",
+      " 7   retweets_count  417 non-null    int64 \n",
+      " 8   likes_count     417 non-null    int64 \n",
+      "dtypes: int64(5), object(4)\n",
+      "memory usage: 29.4+ KB\n"
+     ]
+    }
+   ],
+   "source": [
+    "df.info()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.6"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```

Comment:
Why do you have both an ipynb and a py version of this? Is it essential that both of these be visible in the project?